assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -311,8 +321,11 @@ export default function EffectChooser({
   );
 
   return (
```

Comment:
How accessible is this whole component actually?

It doesn't look like it's announced as a list or anything.

Worried that we ship something that's not accessible.

Reply:
Apologies for the late response. Overall the SR works pretty well with this, was missing a label for the list though. Added that here. 
![Screen Shot 2020-12-11 at 4 41 56 PM](https://user-images.githubusercontent.com/10720454/101964885-123a9e80-3bd0-11eb-8332-a91e2e2a43ca.png)